solvebio/cli/main.py

```diff
@@ -260,6 +260,28 @@ class SolveArgumentParser(argparse.ArgumentParser):
                 }
             ]
         },
+        'download': {
+            'func': data.download,
+            'help': 'Download a file or directory from a SolveBio Vault. Not recursive '
+                    '(will download only files in specified folder).',
+            'arguments': [
+                {
+                    'flags': '--full-path',
+                    'help': 'The full path where the files are located in SolveBio.',
+                    'action': TildeFixStoreAction,
+                    'default': '~/'
+                },
+                {
+                    'flags': '--query',
+                    'help': 'Can be used to filter only needed files from folder (for example, ".vcf").',
+                },
+                {
+                    'name': 'local_path',
+                    'help': 'The path to the local directory where '
+                            'to download files.',
+                }
+            ]
+        },
```

Comment:
can you add a `--dry-run` mode like the other commands? Dry run would not download the file or create the local directory, but print that it would (see other commands)